pkg/scheduler/frameworkext/interface.go

```diff
@@ -64,7 +64,8 @@ type FrameworkExtender interface {
 	// DEPRECATED: use RunReservationExtensionRestoreReservation instead.
 	RunReservationExtensionFinalRestoreReservation(ctx context.Context, cycleState *framework.CycleState, pod *corev1.Pod, states PluginToNodeReservationRestoreStates) *framework.Status
 
-	RunReservationFilterPlugins(ctx context.Context, cycleState *framework.CycleState, pod *corev1.Pod, reservationInfo *ReservationInfo, nodeName string) *framework.Status
+	RunReservationFilterPlugins(ctx context.Context, cycleState *framework.CycleState, pod *corev1.Pod, reservationInfo *ReservationInfo, nodeInfo *framework.NodeInfo) *framework.Status
```

Comment:
Whether we should modifiy some plugin filter to avoid aome redundant Reservation Filter Logic

Reply:
Yes, but in another PR.
For example, the core logic of the nodenumaresource plugin and the deviceshare plugin, i.e. tryAllocateFromReservation, includes both Filter and Allocate logic, so it would take more work if we want to refactor them cleanly.